duckduckgo/Android
# Update app icon, text and colors

Asana Issue URL: 
https://app.asana.com/0/488551667048375/478744499118220 (although much of it was done by other tickets)

## Description
Update app icon, text and colors

## Steps to Test this PR:
1. Ensure app icon is a ddg icon (it was previously the default android icon)
1. Ensure that the title bar is now a very dark grey (primaryDark color was previously blue)
1. Ensure that the progress bar is now orange (accent color was previously pink)
